.github/workflows/discord-pr.yml

```
@@ -0,0 +1,128 @@
+      YELLOW: "16776960"
+
+    steps:
+      - name: Ensure curl & jq
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y jq
+
+      # === PR 이벤트 알림 ===
+      - name: Notify on PR events
+        if: ${{ github.event_name == 'pull_request' }}
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)